pkg/common/crypto/api.go

```diff
@@ -0,0 +1,12 @@
+package crypto
+
+import "github.com/hyperledger/aries-framework-go/pkg/framework/spi/crypto"
+
+// package Crypto contains the consolidated Crypto interface to be used by the framework.
+// it will be created via Options creation of pkg/framework/provider/crypto (implementation of
+// pkg/framework/spi/crypto)
+
+// FrameworkCrypto interface provides all crypto operations needed in the Aries framework
+type FrameworkCrypto interface {
```

Comment:
We don't need the prefix `Framework`. This is implied and this prefix hasn't been used in other interfaces.

Reply:
done